src/Lens/VaultLens.sol

```diff
@@ -12,17 +13,25 @@ import {Utils} from "./Utils.sol";
 import "evk/EVault/shared/types/AmountCap.sol";
 import "./LensTypes.sol";
 
-contract VaultLens is Utils {
+contract VaultLens is Ownable, Utils {
     address internal constant USD = address(840);
 
-    OracleLens public immutable oracleLens;
-    IRMLens public immutable irmLens;
+    OracleLens public oracleLens;
+    IRMLens public irmLens;
 
-    constructor(address _oracleLens, address _irmLens) {
+    constructor(address _owner, address _oracleLens, address _irmLens) Ownable(_owner) {
```

Comment:
If this contract is now Ownable (and therefore in the UI trust-radius) shouldn't we refer to it in the `script/production/mainnet/OwnershipTransferPeriphery.s.sol` script too?

Reply:
reverting that change